gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Sun Oct 23 16:14:38 PDT 2016
+#Tue Jul 25 15:15:03 IST 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.14.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-3.3-all.zip
```

Comment:
1. Is it absolutely necessary to upgrade this version?
2. I am assuming developers will need to upgrade these tools for their development, is this right?
3. Will the end users need to do any android updates or system updates in order to run this new app?